code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -513,12 +513,22 @@ \subsubsection{Theoretical Models}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           \displaystyle\sum{{F_{\text{x}}}}=0\land{}\displaystyle\sum{{F_{\text{y}}}}=0\land{}\displaystyle\sum{M}=0
+           \displaystyle\sum{{F_{\text{x}}}}=0
+           \end{displaymath}
+           \begin{displaymath}
+           \displaystyle\sum{{F_{\text{y}}}}=0
+           \end{displaymath}
+           \begin{displaymath}
+           \displaystyle\sum{M}=0
```

Comment:
All are displayed on new lines, but I don't know if this is a standard or if I should be adding a HARDNL between them.

Reply:
`displaymath` are automatically displayed on new lines, so it's good as is.

Reply:
That's good to know. Thank you!